### ISSUE TITLE
Parameter issue with hugsql snippets

### ISSUE THREAD

**Issue by kanwei** [igrishaev/pg2#25]
Our query:

```
-- :name get-donation-records-from-query :?
SELECT t.*
FROM (
:snip:main-donations-query
) t

ORDER BY :sql:order-col :sql:order-dir, donation_id DESC
LIMIT :limit
OFFSET :offset;
```

:snip:main-donations-query is a snippet that takes one param.

When we run ```(db/get-donation-records-from-query-sqlvec query*)```

We get

```
["SELECT t.*
  FROM (
  SELECT * FROM donation
  WHERE donation.account_id IN ($1)
  ) t
  
  ORDER BY created_at ASC, donation_id DESC
  LIMIT $1
  OFFSET $2;"
 6
 nil
 nil]
```

Note that there's two $1, meaning the parameters from the hugsql :snip aren't considered as separate. Thus, the error is org.pg.error.PGError: Wrong parameters count: 3 (must be 2)

**Comment by igrishaev** [comment]
Hi @kanwei ! I believe this was fixed in `[0.1.24]`, could you please give it a try?

**Comment by igrishaev** [comment]
Closing as it has been tested locally. Feel free to reopen if you find any problems.
